src/applications/gi/containers/Modals.jsx

```diff
@@ -4,703 +4,662 @@ import * as actions from '../actions';
 import Modal from '@department-of-veterans-affairs/formation-react/Modal';
 
 export class Modals extends React.Component {
-  constructor(props) {
-    super(props);
-    this.shouldDisplayModal = this.shouldDisplayModal.bind(this);
```

Comment:
I like this! It inflates the size of the PR, but I dig the readability improvement. :+1: